libs/README.rst

```diff
@@ -76,13 +76,30 @@ notwendig.)
 macOS
 ^^^^^
 Unter MacOS ist die Einrichtung relativ einfach und bedarf nur der oben genannten Voreinstellungen.
-Es wird der von Apple ausgelieferte clang compiler verwendet.
+Es wird der von Apple ausgelieferte clang compiler verwendet. 
+ Beispiel: Innerhalb von macOS /Users/governikus/AusweisApp2 befindet sich der Quellcode.
+::
+
+   $ cd /Users/governikus
+   $ mkdir build
+   $ cd build
+   $ cmake -DCMAKE_BUILD_TYPE=release ../AusweisApp2/libs
+   $ make
 
-Beispiel: Innerhalb von /Users/governikus/AusweisApp2 befindet sich der Quellcode.
 
+Linux / Unix
+^^^^^^^^^^^^
+Die Einrichtung unter Linux/Unix ist einfach und erfordert nur die oben genannten Einstellungen. Die meisten Distributionen haben den C Compiler bereits vorinstalliert, aber er kann auch nachträglich installiert werden.
+ Beispiel: Innerhalb von Linux/Unix /home/governikus/AusweisApp2 befindet sich der Quellcode
+ 
+ Install dependencies : 
+ 
+` sudo apt install  libpcsclite1 libpcsclite-dev qttools5-dev libqt5svg5-dev libqt5websockets5-dev libqt5qml5  libqt5qml5-dev qtdeclarative5-dev qtbase5-dev  qt-quickcontrols2-5-dev
```

Comment:
no need for libpcsclite1 and libqt5qml5 - they are already dependencies of their `-dev` variants

Reply:
QT5 is outdated in the meantime and QT6 (currently used 6.5.3 with 2.0.1) is needed.

Reply:
Maybe its also worth to mention to which distro and version it belongs to.

Reply:
Sorry, we won't add any distribution specific packages here. This is outdated REALLY FAST. Also this is a job for the package maintainer of the linux distribution.